addons/sale_management/views/sale_order_template_views.xml

```diff
@@ -115,9 +115,11 @@
                             <field name="mail_template_id" context="{'default_model':'sale.order'}"/>
                         </group>
                     </page>
-                    </notebook>
-                    <field name="note" nolabel="1"
+                    <page string="Terms &amp; Conditions" name="terms_and_conditions">
+                        <field name="note" nolabel="1"
                         placeholder="The Administrator can set default Terms &amp; Conditions in Sales Settings. Terms set here will show up instead if you select this quotation template."/>
```

Comment:
As you added a tab in the line above, you should add a tab here as well to keep the indentation